preset/src/shortcuts/slider.ts

```diff
@@ -0,0 +1,9 @@
+export const sliderShortcuts: Record<string, string> = {
+    // select
```

Comment:
`preset/src/shortcuts/common.ts` contains shortcuts for some class combinations, you can use them, such as `flex items-center justify-center -> fcc`

Reply:
oh my oversight